packages/terra-collapse/tests/nightwatch/CollapseTests.jsx

```diff
@@ -0,0 +1,20 @@
+/* eslint-disable import/no-extraneous-dependencies */
+
+import React from 'react';
+import { Link } from 'react-router';
+
+const CollapseTests = () => (
+  <div>
+    <ul>
+      <li><Link to="/tests/collapse-tests/default">Collapse - Default</Link></li>
+      <li><Link to="/tests/collapse-tests/customized-button">Collapse - Custom Button</Link></li>
+      <li><Link to="/tests/collapse-tests/closed-button-text">Collapse - Custom Close Button Text</Link></li>
```

Comment:
For `Custom Close Button Text` should it be changing the button text on open vs close?